inputfiles/addedTypes.json

```diff
@@ -1437,5 +1437,11 @@
             "name": "reason?",
             "type": "any"
         }]
+    },
+    {
+        "kind": "method",
+        "interface": "Body",
```

Comment:
Thanks, I added `"flavor": "Web"`and generated *.d.ts files. Now the CI have passed.